CMakeLists.txt

```diff
@@ -324,6 +324,23 @@ if(NOT MSVC OR ENABLE_VSTEST)
 	add_subdirectory(tests)
 endif()
 
+if(NOT MSVC)
+	# Create pkgconfig files.
+	set(prefix      ${CMAKE_INSTALL_PREFIX})
+	set(exec_prefix \${prefix})
+	set(libdir      ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
```

Comment:
Hi,
This line should be `set(libdir \${exec_prefix}/lib`, no ?
Only libdir doesn't use variable to define its path.
Are there any reasons to use CMAKE_* variables ?

Reply:
Whoops. No there isn't.

Reply:
Keeping ${CMAKE_INSTALL_LIBDIR} though so it doesn't break lib64 systems.

Reply:
Thanks for clarification. I understand. I think both resolutions for pkgconfig and pqueue.h are good.